bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieClient.java

```diff
@@ -98,7 +101,13 @@ public BookieClient(ClientConfiguration conf, EventLoopGroup eventLoopGroup,
     }
 
     public BookieClient(ClientConfiguration conf, EventLoopGroup eventLoopGroup,
-                        OrderedSafeExecutor executor, StatsLogger statsLogger) throws IOException {
+                        OrderedSafeExecutor executor, ScheduledExecutorService scheduler) throws IOException {
```

Comment:
who is using this constructor?

Reply:
+1 for dropping if not used in BK neither in DL

Reply:
killed all constructors except the main one.